File: VideoEditorText/src/main/java/com/imagevideoeditor/MainActivity.java
package com.imagevideoeditor;

import android.content.Intent;
import android.os.Build;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.annotation.NonNull;
import androidx.annotation.RequiresApi;
import androidx.appcompat.app.AppCompatActivity;

import com.imagevideoeditor.Utils.CameraUtils;
import com.kbeanie.multipicker.api.CameraVideoPicker;
import com.kbeanie.multipicker.api.entity.ChosenImage;
import com.kbeanie.multipicker.api.entity.ChosenVideo;

import java.util.List;

public class MainActivity extends AppCompatActivity implements CameraUtils.OnCameraResult {

    private CameraUtils cameraUtils;
    private CameraVideoPicker cameraVideoPicker;
    Button btnPhoto , btnVideo ;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_video_text);
        cameraUtils = new CameraUtils(this, this);

        btnVideo = findViewById(R.id.btnVideo);
        btnPhoto = findViewById(R.id.btnPhoto);

        btnPhoto.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
//                cameraUtils.openCameraGallery();
            }
        });
        btnVideo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                cameraUtils.alertVideoSelcetion();
            }
        });
    }

    @RequiresApi(api = Build.VERSION_CODES.M)
    @Override
    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
        super.onRequestPermissionsResult(requestCode, permissions, grantResults);

        cameraUtils.onRequestPermissionsResult(requestCode, permissions, grantResults);
    }

    @Override
    public void onSuccess(List<ChosenImage> images) {
        if (images != null && images.size() > 0) {
            Intent i = new Intent(MainActivity.this, PreviewPhotoActivity.class);
            i.putExtra("DATA", images.get(0).getOriginalPath());
            //binding.ivProfilePic.setImageURI(Uri.fromFile(selectedImageFile));
            startActivity(i);

        }
    }

    @Override
    public void onError(String error) {

    }

    @Override
    public void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);
        cameraUtils.onActivityResult(requestCode, resultCode, data);

    }


    @Override
    public void onVideoSuccess(List<ChosenVideo> list) {
        if (list != null && list.size() > 0) {
            Intent i = new Intent(MainActivity.this, PreviewVideoActivity.class);
            i.putExtra("DATA", list.get(0).getOriginalPath());
            //binding.ivProfilePic.setImageURI(Uri.fromFile(selectedImageFile));
            startActivity(i);

        }
    }
}

File: VideoEditorStcikers/src/main/java/com/imagevideoeditor/MainActivity.kt
package com.imagevideoeditor

import android.content.Intent
import android.os.Build
import android.os.Bundle
import androidx.annotation.RequiresApi
import androidx.appcompat.app.AppCompatActivity
import com.imagevideoeditor.Utils.CameraUtils
import com.imagevideoeditor.Utils.CameraUtils.OnCameraResult
import com.kbeanie.multipicker.api.CameraVideoPicker
import com.kbeanie.multipicker.api.entity.ChosenImage
import com.kbeanie.multipicker.api.entity.ChosenVideo
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity(), OnCameraResult {
    private var cameraUtils: CameraUtils? = null
    private val cameraVideoPicker: CameraVideoPicker? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        cameraUtils = CameraUtils(this, this)
        btnPhoto!!.setOnClickListener {
            //                cameraUtils.openCameraGallery();
        }
        btnVideo!!.setOnClickListener { cameraUtils!!.alertVideoSelcetion() }
    }

    @RequiresApi(api = Build.VERSION_CODES.M)
    override fun onRequestPermissionsResult(
        requestCode: Int,
        permissions: Array<String>,
        grantResults: IntArray
    ) {
        super.onRequestPermissionsResult(requestCode, permissions, grantResults)
        cameraUtils!!.onRequestPermissionsResult(requestCode, permissions, grantResults)
    }

    override fun onSuccess(images: List<ChosenImage>) {
        if (images != null && images.size > 0) {
            val i = Intent(this@MainActivity, PreviewPhotoActivity::class.java)
            i.putExtra("DATA", images[0].originalPath)
            //binding.ivProfilePic.setImageURI(Uri.fromFile(selectedImageFile));
            startActivity(i)
        }
    }

    override fun onError(error: String) {}
    public override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
        super.onActivityResult(requestCode, resultCode, data)
        cameraUtils!!.onActivityResult(requestCode, resultCode, data)
    }

    override fun onVideoSuccess(list: List<ChosenVideo>) {
        if (list != null && list.size > 0) {
            val i = Intent(this@MainActivity, PreviewVideoActivity::class.java)
            i.putExtra("DATA", list[0].originalPath)
            //binding.ivProfilePic.setImageURI(Uri.fromFile(selectedImageFile));
            startActivity(i)
        }
    }
}